server.js

```diff
@@ -1,33 +1,73 @@
-import express from 'express'
+import express, { response } from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import booksData from './data/books.json'
 
 // Defines the port the app will run on. Defaults to 8080, but can be 
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
 const port = process.env.PORT || 8080
-const app = express()
+const app = express() // a variable that allows us to create a server
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// accepting get requests to this endpoint, / means no path after the address
+app.get('/', (request, response) => {
+  response.send('Welcome to my books API // Anna Hellqvist 📚')
+})
```

Comment:
[Here](https://www.npmjs.com/package/express-list-endpoints)'s a cool lib that you can try out next time if you'd like to include some documentation in your API!

Reply:
Thank you, I will check it out! ✨